qirlib/src/evaluation/jit.rs

```diff
@@ -116,7 +118,7 @@ fn is_entry_point(function: FunctionValue) -> bool {
         .is_some()
 }
 
-fn module_functions<'ctx>(module: &Module<'ctx>) -> impl Iterator<Item = FunctionValue<'ctx>> {
+pub fn module_functions<'ctx>(module: &Module<'ctx>) -> impl Iterator<Item = FunctionValue<'ctx>> {
```

Comment:
Should probably leave this as `pub(crate)` for now. I'm not sure we want to make this externally visible yet.

Reply:
Good point, done in 76361cc.